run_trial.bash

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+#TODO: RE-ENABLE THIS. (dhood): it was disabled because when the server container shuts down, it appears to return non-zero.
+# set -e
+
+TEAM_NAME=$1
+TRIAL_NAME=$2
+
+#TODO: generate unique container names based on input arguments
+CONTAINER_NAME=ariac-server-system
+
+HOST_LOG_DIR=`pwd`/logs/${TEAM_NAME}/${TRIAL_NAME}
+echo "Creating directory: ${HOST_LOG_DIR}"
+mkdir -p ${HOST_LOG_DIR}
+
+# TODO: don't rely on script being run in the root directory
+# TODO: error checking for case when files can't be found
+TEAM_CONFIG_DIR=`pwd`/${TEAM_NAME}
+echo "Using team config: ${TEAM_CONFIG_DIR}/team_config.yaml"
+COMP_CONFIG_DIR=`pwd`/comp_configs
+echo "Using comp config: ${COMP_CONFIG_DIR}/${TRIAL_NAME}.yaml"
+
+LOG_DIR=/ariac/logs
+
+# Create the network for the containers to talk to each other.
+./ariac-competitor/ariac_network.bash
+
+# Start the competitors container and let it run in the background.
+#TODO: parameterize the container name
+./ariac-competitor/run_competitor_container.bash &
```

Comment:
This background call to run_competitor_container script is going to be hard to check and response on errors. For example I forget to generate the team container and run the script. It keeps going even if this call fails totally which ends up in other errors more difficult to debug. We can workaround with some checks before.